tools/taskcluster-proxy/routes.go

```diff
@@ -79,6 +79,22 @@ func (routes *Routes) setHeaders(res http.ResponseWriter) {
 	}
 }
 
+// Implement the Handler interface, dispatching requests to the various
+// *Handler methods.  Note that we cannot use ServeMux for this as it mangles
+// URLs and sends redircts.
+func (routes *Routes) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	url := r.URL.String()
+	if strings.HasPrefix(url, "/bewit") {
```

Comment:
will this handle (and do we want to handle) a situation where this is `"//bewit"`, etc?

Reply:
I don't think we want to handle `http://taskcluster//bewit`, unless I'm misunderstanding the question.